### DISCUSSION TITLE
mimir docs suggest using etcd-operator for the HA tracker but etcd-operator is archived

### DISCUSSION THREAD

**Discussion by jymmi** [grafana/mimir#5339] [Help and support]
on https://grafana.com/docs/mimir/latest/operators-guide/deploy-grafana-mimir/jsonnet/deploy/ there is a suggestion to install/use the etcd-operator for use with the ha tracker.  The jsonnet provides a kind:EtcdCluster to spawn the cluster.  Unfortunately the etcd-operator has been archived since 3/27/2020 and is no longer maintained.

https://github.com/coreos/etcd-operator/tree/master
`This repository has been archived by the owner on Mar 27, 2020. It is now read-only.`

This means in its current form it will not install in newer k8s clusters which have deprecated many v1beta1 apis.  What is the suggested path forward for using the HA tracker with etcd?  

- fork etcd-operator and update to v1 api? 
- manually deploy an etcd cluster?  
- something else?

How has grafana labs dealt with this on their end? 

Many thanks.

**Comment by sarita-maersk** [comment]
@jymmi  could you figure out what is the way forward? I am also struggling.
